src/library_wget.js

```diff
@@ -175,9 +175,7 @@ var LibraryWget = {
       if (onerror) {
         var statusText = 0;
         if (http.statusText) {
-          var len = lengthBytesUTF8(http.statusText) + 1;
-          statusText = stackAlloc(len);
-          stringToUTF8(http.statusText, statusText, len);
+          statusText = allocateUTF8OnStack(http.statusText);
```

Comment:
This looks like it's missing a `withStackSave`.

Reply:
Fixed.